tools/documentation/deployment-checklist.md

```diff
@@ -0,0 +1,116 @@
+## General
+Most of this can only be done by repository administrators, who are members of the `openscope-admins` team on GitHub, and have access to the openScope app pipeline on Heroku.
+
+## Heroku Apps
+There are four types of apps we use on Heroku: the [`production`](https://openscope-prod.herokuapp.com/) app, the [`staging`](https://openscope-staging.herokuapp.com/) app, the [`development`](https://openscope-dev.herokuapp.com/) app, and then review apps admins can very easily spool up for individual feature branches. With all of these apps, when new code is pushed to their respective branch on the openScope repository, the app will automatically begin to rebuild. This takes about five minutes, and will re-deploy if the build succeeds. You can tell when the process has completed because a message will be displayed in slack in the `dev` channel, showing the results of the build.
+
+The __production__ app is the exact same one that the website uses. When you visit [www.openscope.co](http://www.openscope.co), you are actually interacting with this app through Heroku. We want to keep this app as reliable and pristine as possible.
+
+The __staging__ app is only in use briefly, at the end of each sprint. It represents what the production app _will be_ after work from the current sprint is merged. It is vital that we diligently test this app for a few days before pushing the changes to production, because we want to prevent exposing our regular users to annoying bugs that we could have caught before releasing.
+
+The __development__ app is where we merge all features and bugfixes during the development phase of the sprint. Occasionally, things will break here, which is a fact of life. At the end of the sprint, we review, and fix anything that may have inadvertently been broken.
+
+The __review__ apps can be created (by admins only) at the click of a button for any open pull request. This can be very useful if you are not set up to build and host your app locally on your computer, because you can still continue to see the results of your changes.
```

Comment:
we can add something like: This can also be helpful for those times a specific branch needs to be shared with the wider development group when heavier testing is required/desired before merging in to `develop`